### ISSUE TITLE
Savon does not use wsdl namespace identifiers

### ISSUE THREAD

**Issue by atrzaska** [savonrb/savon#954]
# Bug report

**Current behavior:**

Savon does not read wsdl namespace identifiers.

It generates following message body:
```
<?xml version="1.0" encoding="UTF-8"?>
<env:Envelope
	xmlns:xsd="http://www.w3.org/2001/XMLSchema"
	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
	xmlns:tns="http://servlets.iex.jv.com/"
	xmlns:env="http://schemas.xmlsoap.org/soap/envelope/">
	<env:Body>
		<extractToXML>
                     ....
		</extractToXML>
	</env:Body>
</env:Envelope>
```

This does not work with the endpoint which requires `tns` namespace. WSDL is parsed correctly but namespace is skipped. 
```
<?xml version="1.0" encoding="UTF-8"?>
<env:Envelope
	xmlns:xsd="http://www.w3.org/2001/XMLSchema"
	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
	xmlns:tns="http://servlets.iex.jv.com/"
	xmlns:env="http://schemas.xmlsoap.org/soap/envelope/">
	<env:Body>
		<tns:extractToXML>
                     ....
		</tns:extractToXML>
	</env:Body>
</env:Envelope>
```
After applying this fix the endpoint worked fine:

<img width="1331" alt="Screenshot 2021-04-09 at 18 31 21" src="https://user-images.githubusercontent.com/2529820/114212677-62e5bc80-9962-11eb-85ca-d2337841f676.png">

**Steps to reproduce current behavior:**

```
client = Savon.client(wsdl: 'https://cvlizer.joinvision.com/cvlizer/exservicesoap?wsdl')
message = {
  username: 'foo',
  password: 'foo',
  language: 'en',
  model: 'jobolizer_3_0',
  inputdatatype: 'url',
  inputdata: Base64.encode64('https://google.com')
}
client.call(:extract_to_xml, message: message)
```

I had to fiter the credentials because they are confidential.

**Expected behavior:**

Following XML message should be genrated with correct namespace of the message:

```
<?xml version="1.0" encoding="UTF-8"?>
<env:Envelope
	xmlns:xsd="http://www.w3.org/2001/XMLSchema"
	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
	xmlns:tns="http://servlets.iex.jv.com/"
	xmlns:env="http://schemas.xmlsoap.org/soap/envelope/">
	<env:Body>
		<tns:extractToXML>
                     ....
		</tns:extractToXML>
	</env:Body>
</env:Envelope>
```

**System information:**

* ruby version: 3
* savon version: 2.12.1

**Comment by edlebert** [comment]
I'm also seeing this, but only after upgrading from wasabi 3.5.1 to 3.6.1 (latest).

**Comment by nbarrientos** [comment]
Just hit the same issue, too. The suggested patch works.

**Comment by nbarrientos** [comment]
Downgrading Wasabi to 3.5.1 also helps indeed.

**Comment by Sh1ken** [comment]
Just experienced this bug and downgrading Wasabi to 3.5.0 by itself worked as expected.

**Comment by SleeplessByte** [comment]
It's not stale.

**Comment by pcai** [comment]
I believe this was fixed in https://github.com/savonrb/wasabi/pull/103 which was released in wasabi 3.7.0. Can you please reopen with repro information if this is still not working?
